src/common/playcreated/play-created.scss

```diff
@@ -21,7 +21,6 @@
       content: "";
       position: absolute;
       height: 4px;
-      background-color: var(--color-brand-primary);
```

Comment:
@Hat52 , Why are we removing this?

Reply:
you can look at the related #703 issue to get better understanding.

Reply:
This is the change that I have done on another PR.